tests/ImageSharp.Tests/Image/ImageTests.WrapMemory.cs

```diff
@@ -173,6 +215,73 @@ public void WrapSystemDrawingBitmap_WhenOwned()
                 }
             }
 
+            [Fact]
+            public void WrapMemory_FromBytes_CreatedImageIsCorrect()
+            {
+                Configuration cfg = Configuration.Default.Clone();
```

Comment:
Better call `Configuration.CreateDefault()`

Reply:
I actually had copied that from the other existing test 😄

I proceeded to update the other one too while I was it then, done in https://github.com/SixLabors/ImageSharp/pull/1314/commits/ff5e3015a0567858195841f8941ee8b771189f18.